docs(topology): update README.md (#678)
janus-idp/backstage-plugins: plugins/topology/README.md

- The following annotations are added to workload resources, such as Deployments to navigate to the Git repository of the associated application using the edit code decorator:

  ```yaml title="deployment.yaml"
  annotations:
    app.openshift.io/vcs-uri: <GIT_REPO_URL>
  ```

  You can also add the following annotation to navigate to a specific branch:

  ```yaml title="deployment.yaml"
  annotations:
    app.openshift.io/vcs-ref: <GIT_REPO_BRANCH>
  ```

- The following label is added to workload resources, such as Deployments to display runtime icon in the topology nodes:

  ```yaml title="deployment.yaml"
  labels:
    app.openshift.io/runtime: <RUNTIME_NAME>
  ```

  As another option, you can include the following label to display the runtime icon:

  ```yaml title="deployment.yaml"
  labels:
    app.kubernetes.io/name: <RUNTIME_NAME>
  ```

>   Currently supported values of `<RUNTIME_NAME>`:

Reviewer: ```suggestion
  Supported values of `<RUNTIME_NAME>` include:
```

  - django
  - dotnet
  - drupal
  - go-gopher
  - golang
  - grails
  - jboss
  - jruby
  - js
  - nginx
  - nodejs
  - openjdk
  - perl
  - phalcon
  - php
  - python
  - quarkus
  - rails
  - redis
  - rh-spring-boot
  - rust
  - java
  - rh-openjdk
  - ruby
  - spring
  - spring-boot

>   Any other values would result in no icons being rendered for the node.

Reviewer: ```suggestion
Other values result in icons not being rendered for the node. 
 
```
